classes/page_group_table.php

```diff
@@ -71,7 +82,8 @@ public function setup() {
         $this->set_sql(
             '*',
             '{' . page_group::TABLE . '}',
-            '1=1'
+            $this->month ? 'month = ?' : '1=1',
+            $this->month ? [$this->month] : []
```

Comment:
Can we move this into a previous prepare block and do something if the month is set, and do something else if the month is not set? This would keep it more consistent with wider usage and will be easier to read.

e.g.
```
// Defaults ...

// When month is set.
if ($this->month) {
    ...
}

$this->set_sql(
    '*', 
    '{' . page_group::TABLE . '}',
    $where,
    $params
);
```

Reply:
Also to reiterate the previous comment, would $this->month ever be null (in practice)?

Reply:
I've refactored that there will always be a month set.